#message to represent a path generated by path planner.each segment of which is a dubins curve with a stop point

DubinSeg_msg[] dubin_path
